core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
on my last bit I was saying I realized I was basing my argument partially on an incorrect statement although I still prefer keeping things in config format instead of hardcoded here and there in the code.

That link at the top was something I saw which seemed like an interesting middle ground between the two if one really wanted to try to obscure "dangerous" bits from a user, although it has some of the same issues as putting stuff in non-standard places.